### PR TITLE
Features/home page

### PR DESCRIPTION
**Pull request by SHIKHASINGH1506**
Added featured category in home page

### CODE REVIEW

File: Components/Home/home.css [hunk 1]
```diff
@@ -22,7 +29,7 @@
   border-radius: 0.75rem;
 }
 
-@media only screen and (max-width: 640px) {
+@media only screen and (max-width: 768px) {
```
Comment (HarshnaNHaswani) [inline Components/Home/home.css:29]: can use variables for media queries

File: index.html [hunk 2]
```diff
@@ -34,7 +34,8 @@
               <rect y="20" width="60" height="10"></rect>
```
Comment (HarshnaNHaswani) [inline index.html:34]: avoid inline styles

File: Components/Navbar/navbar.css [hunk 3]
```diff
@@ -177,7 +176,7 @@
 /* media query */
 @media screen and (max-width:426px){
   .text-overlay .text-overlay-title{
-    font-size: 1rem;
+    font-size: 25.2px;
```
Comment (HarshnaNHaswani) [inline Components/Navbar/navbar.css]: use responsive unit, media queries should be in ems

File: Components/Home/home.css [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+.asideSection {
+  margin-top: 1rem;
+  display: grid;
+  padding: 0 4rem;
+  grid-template-columns: repeat(auto-fit, minmax(15%, 1fr));
+}
+.feature-box {
+  position: relative;
+  padding: 0 1rem;
+  max-width: 600px;
```
Comment (HarshnaNHaswani) [inline Components/Home/home.css]: change units

**Review by HarshnaNHaswani — COMMENTED** [review]
Few points that I noted/ suggestions:

- really pleasant ambience of the site, clean easy-to-use, everything is clearly understood and accessible
- the features displayed on the mobile view in home page can be shown in desktop view too, maybe as text on image
- different name for site; zia everywhere, FERN on header(see index.html)
- Your wishlist product name is "Appliqued Crew-Neck T-shirt"

File: Components/Home/home.css [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
```
Comment (bishnoimukesh) [inline Components/Home/home.css]: use single unit throughout

File: Components/Navbar/navbar.css (hunk 3 above)
Comment (bishnoimukesh) [inline Components/Navbar/navbar.css]: same

File: index.html [hunk 6]
```diff
@@ -5,8 +5,11 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>ZIA</title>
-    <link rel="stylesheet" href="https://kube-ui.netlify.app/Components/kube-ui.css" />
-    <link rel="stylesheet" href="https://kube-ui.netlify.app/utility.css">
+    <link
+      rel="stylesheet"
+      href="https://kube-ui.netlify.app/Components/kube-ui.css"
+    />
```
Comment (bishnoimukesh) [inline index.html:11]: same link showing changes

File: index.html [hunk 7]
```diff
@@ -31,13 +34,18 @@
...
               <li class="nav-item">
                 <a class="nav-item-link" href="">HOME</a>
               </li>
               <li class="nav-item">
-                <a class="nav-item-link" href="./Components/Products/product.html">PRODUCT</a>
+                <a
+                  class="nav-item-link"
+                  href="./Components/Products/product.html"
+                  >PRODUCT</a
+                >
```
Comment (bishnoimukesh) [inline index.html:48]: avoid this as it is showing changes

File: index.html [hunk 8]
```diff
@@ -55,17 +63,25 @@
             </div>
             <ul class="navbar-nav navbar-fixed">
               <li class="nav-item">
-                <a class="nav-item-link" href="./Components/Login/login.html">LOGIN</a>
+                <a class="nav-item-link" href="./Components/Login/login.html"
+                  >LOGIN</a
+                >
```
Comment (bishnoimukesh) [inline index.html:68]: same

File: index.html [hunk 9]
```diff
@@ -55,17 +63,25 @@
...
               </li>
               <li class="nav-item">
-                <a class="nav-icon-link badge-container no-link-style" href="./Components/Cart/cart.html">
+                <a
+                  class="nav-icon-link badge-container no-link-style"
+                  href="./Components/Cart/cart.html"
+                >
```
Comment (bishnoimukesh) [inline index.html:74]: same

**Review by bishnoimukesh — COMMENTED** [review]
some suggestions are given
rest looks good

**Comment by SHIKHASINGH1506** [comment]
Changes pushed as per PR review suggestions.

**Pull request merged**